frontend/awx/main/routes/useAwxOrganizationsRoutes.tsx

```diff
@@ -41,14 +43,14 @@ export function useAwxOrganizationRoutes() {
               element: <OrganizationDetails />,
             },
             {
-              id: AwxRoute.OrganizationUsers,
+              id: AwxRoute.OrganizationUsersAccess,
               path: 'users',
-              element: <OrganizationAccess />,
+              element: <OrganizationUserAccess />,
             },
             {
-              id: AwxRoute.OrganizationTeams,
+              id: AwxRoute.OrganizationTeamsAccess,
               path: 'teams',
```

Comment:
We are trying to unify url paths to use `team-access`. Could you change it, please?

Reply:
same comment at [above](https://github.com/ansible/ansible-ui/pull/2362#discussion_r1617507202). This path matches what we have downstream for Organization -> Teams.